sceptre/plan/actions.py

```diff
@@ -677,14 +677,21 @@ def estimate_cost(self):
         )
         return response
 
-    def stack_name(self):
+    def stack_name(self, p_opt):
         """
         Returns the Stack's stack name.
 
+        :param p_opt: The -P option.
+        :type p_opt: bool
```

Comment:
`p_opt`is not a very descriptive variable name

Reply:
ok changed it to `print_name`.